src/ray/protobuf/gcs.proto

```diff
@@ -42,11 +42,13 @@ enum TablePrefix {
   // WORKER is already used in WorkerType, so use WORKERS here.
   WORKERS = 15;
   INTERNAL_CONFIG = 16;
-  TABLE_PREFIX_MAX = 17;
-  PLACEMENT_GROUP_SCHEDULE = 18;
-  PLACEMENT_GROUP = 19;
-  KV = 20;
-  ACTOR_TASK_SPEC = 21;
+  PLACEMENT_GROUP_SCHEDULE = 17;
+  PLACEMENT_GROUP = 18;
+  KV = 19;
+  ACTOR_TASK_SPEC = 20;
+  // NOTE: This is the placeholder for the enumeration number.
+  // It must be placed at the end of TablePrefix.
+  TABLE_PREFIX_MAX = 21;
```

Comment:
Is it used any place? Should we just remove this? If we'd like to get max value of this enum, I think there is a grpc API for this.

Reply:
This field is indeed unused, I'am ok to delete it.